tests/integration/socialAuth/twitter.test.js

```diff
@@ -0,0 +1,55 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import {
+  mockSocialAuth,
+  mockSocialAuthNonUser,
+  dummyUser
+} from "../../helpers/socialAuthHelpers";
+import app from "../../../index";
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+describe.skip("mocking social authentication with twitter", () => {
+  mockSocialAuth("https://api.twitter.com/");
+  it("should redirect a user to profile page with data", done => {
+    chai
+      .request(app)
+      .get("/api/v1/auth/twitter")
+      .end((err, res) => {
+        expect(res.redirect).to.be.equals(true);
+        expect(res.status).to.be.equal(302);
+        expect(res.header.location).to.be.equals(
+          `/api/v1/profile/${dummyUser.username}`
+        );
+        done();
+      });
+  });
```

Comment:
@espoirMur For the sake of consistency, we agreed on adding space after each test case. Therefore, you can add a space after each test case.